templates/taxi/driver_detail.html

```diff
@@ -2,12 +2,23 @@
 
 {% block content %}
   <h1>
-    Username: {{ driver.username }}
+    Username: {{ driver.username }} 
+    <a href="{% url 'taxi:driver-delete' driver.id %}" class="btn btn-danger">
+      Delete Driver
+    </a>
   </h1>
 
   <p><strong>First name:</strong> {{ driver.first_name }}</p>
   <p><strong>Last name:</strong> {{ driver.last_name }}</p>
-  <p><strong>License number:</strong> {{ driver.license_number }}</p>
+  <p>
+    <strong>License number:</strong> {{ driver.license_number }}
+    <a 
+        href="{% url 'taxi:driver-update' pk=driver.id %}"
```

Comment:
Ensure that the URL tag uses single quotes consistently for better readability. Change `{% url "taxi:driver-update" pk=driver.id %}` to `{% url 'taxi:driver-update' pk=driver.id %}`.